src/elements/Button/Button.vue

```diff
@@ -16,14 +16,6 @@
           fixed-width/>
       </slot>
     </div>
-
-    <div
```

Comment:
The goal was to remove the wrapping `<div>` _while preserving the `<slot>` within_.

Reply:
Sorry for the wrong changes earlier